include/swift/Runtime/Concurrency.h

```diff
@@ -188,6 +188,28 @@ bool swift_task_removeStatusRecord(AsyncTask *task,
 SWIFT_EXPORT_FROM(swift_Concurrency) SWIFT_CC(swift)
 size_t swift_task_getJobFlags(AsyncTask* task);
 
+/// Get a task local value from the passed in task. Its Swift signature is
+///
+/// \code
+/// func _taskLocalGet<Key>(_ task: Builtin.NativeObject, keyType: Key.Type)
+///     -> UnsafeMutableRawPointer? where Key: TaskLocalKey
+/// \endcode
+SWIFT_EXPORT_FROM(swift_Concurrency) SWIFT_CC(swift)
+HeapObject* swift_task_localGet(AsyncTask* task,
```

Comment:
Hmm, okey; not sure exactly about this... for now I can get away with removing the generics from here to be honest I think? Since i need to cast (and i know to which valid) anyway when taking it out again